docs/guides/elements-lookup-troubleshooting.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Elements Lookup Troubleshooting
+---
+
+This article helps to resolve possible issues that may pop up while looking up for elements with XCUITest driver,
+where the desired element is either not found or not visible in the page source at all.
+
+Since there might be multiple reasons to why an element cannot be found the topic is divided into sections where
+each section contains visible symptoms with the list of their possible resolutions.
+
+
+## Symptom #1
```

Comment:
I would actually suggest to have symptom #1 be something like, the desired element is part of a modal or popup drawn over the rest of the application content, but is not present at all in the source tree. The intended solution would be to switch the default active app to `com.apple.springboard`, which is described in `troubleshooting.md`, so it could just be a link to that page section, or perhaps the content from that page could even be moved to this one.

Reply:
added